src/Illuminate/Notifications/Messages/SlackAttachment.php

```diff
@@ -175,6 +189,43 @@ public function fields(array $fields)
     }
 
     /**
+     * Add an action to the attachment.
+     *
+     * @param  \Closure|string  $title
+     * @param  array  $content
+     * @return $this
+     */
+    public function action($title, $content = [])
+    {
+        if (is_callable($title)) {
```

Comment:
I modeled this after the field method on line 166. Does it seem to make sense to refactor both of these? I had a similar thought but decided to model after a method already in the repo.